web/src/assets/styles/blocks.scss

```diff
@@ -196,3 +196,14 @@ section > .content {
     --pf-c-progress--GridGap: var(--spacer-small);
   }
 }
+
+// compact lists in popover
+.pf-c-popover li + li {
```

Comment:
Shouldn't  these stiles go to "patternfly-overrides.scss"?

Reply:
PF defines ".pf-c-list li + li" this uses ".pf-c-popover li + li" so technically it is not an override but rather an extension.